src/chapters/quickconnect.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<chapter xml:id="quickconnect" xmlns="http://docbook.org/ns/docbook" version="5.0" xml:lang="en"
+    xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <title>Quickconnect extension</title>
```

Comment:
Should the title represent more the high-level goal satisfied by the extension, rather than the extension itself? I don't have a suggestion for this off the top of my head, but from the perspective of a user browsing the table of contents for help, they presumably wouldn't know that the solution to their problem is the quickconnect extension until they read the chapter in question.

This is the reasoning behind the existing chapters being named things like "Database authentication" rather than "JDBC extension", or "Proxying Guacamole" rather than "Apache and Nginx".

Reply:
Some ideas:
- Adhoc Connections
- User-defined Connections
- Temporary Connections

I'm having a hard time finding anything that just calls out to me, here...